packages/vite/src/node/optimizer/scan.ts

```diff
@@ -233,6 +233,7 @@ function esbuildScanPlugin(
               if (m.index === importsRE.lastIndex) {
                 importsRE.lastIndex++
               }
+              m[1] = m[1]. replace(/(\r\n|\n|\r)/gm, "")
               js += `\nimport ${m[1]}`
```

Comment:
I think it is better to use a new variable here
```suggestion
              const importPath = m[1].replace(/(\r\n|\n|\r)/gm, "")
              js += `\nimport ${importPath}`
```

Would you add a test case for this? So we can check that the fix is correct and avoid regressions